internal/test_helper.go

```diff
@@ -136,3 +139,54 @@ func PickProcessGroups(cluster *fdbv1beta2.FoundationDBCluster, processClass fdb
 
 	return pickedProcessGroups
 }
+
+// SetupClusterForTest will generate all required resources like Pods for test cases.
+func SetupClusterForTest(cluster *fdbv1beta2.FoundationDBCluster, k8sClient ctrlClient.Client) error {
```

Comment:
mostly just curious, but what is the intent of having this similar function instead of the original `SetupClusterForTest` in `controllers`?

Reply:
cyclic dependencies :)

Reply:
ooh